### PR TITLE
Update to be able to parse both 1.0 and 2.0 jobXML and batchXML files

### PR DESCRIPTION
**Pull request by jhanders34**
- Add xsd files to the container and API bundles
- Update the bundle name to be batch.2.0 instead of batch-2.0
- Add JAXB files for both job V1 and V2 files so that we can read either version of the XML files.
- Add unit test to easily test the ModelResolver.

### CODE REVIEW

**Comment by jhanders34** [comment]
#build

**Comment by jhanders34** [comment]
#libby #build #spawn.fullfat.buckets=com.ibm.ws.jbatch_fat, com.ibm.ws.jbatch.jms_fat, com.ibm.ws.jbatch.rest_fat, com.ibm.ws.jbatch.security_fat, com.ibm.ws.jbatch.utility_fat

**Comment by jhanders34** [comment]
#libby #build #spawn.fullfat.buckets=com.ibm.ws.jbatch_fat, com.ibm.ws.jbatch.jms_fat, com.ibm.ws.jbatch.misc_fat, com.ibm.ws.jbatch.rest_fat, com.ibm.ws.jbatch.security_fat, com.ibm.ws.jbatch.utility_fat

File: dev/com.ibm.jbatch.container/src/com/ibm/jbatch/container/jsl/impl/JobModelHandler.java [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+            final String jaxbPackageName;
+            String schemaName = SCHEMA_LOCATION_V1;
+
+            if ("1.0".equals(version)) {
+                jaxbPackageName = "com.ibm.jbatch.jsl.model.v1";
+                schemaName = SCHEMA_LOCATION_V1;
+            } else {
+                jaxbPackageName = "com.ibm.jbatch.jsl.model.v2";
+                schemaName = SCHEMA_LOCATION_V2;
+            }
```
Comment (cgianfra) [inline dev/com.ibm.jbatch.container/src/com/ibm/jbatch/container/jsl/impl/JobModelHandler.java:117]: So anything not explicitly 1.0 is treated as 2.0? I'm curious what the current behavior is.. does this version attribute not exist, not get evaluated (everything is assumed to be 1.0), or do we actually do something for invalid values?

**Review by cgianfra — APPROVED** [review]

File: dev/com.ibm.jbatch.container/src/com/ibm/jbatch/container/jsl/impl/JobModelHandler.java (hunk 1 above)
Comment (jhanders34) [reply dev/com.ibm.jbatch.container/src/com/ibm/jbatch/container/jsl/impl/JobModelHandler.java:117]: version is a required attribute so this isn't an issue.

**Pull request merged**